content/servicemesh/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Servicemesh with Istio"
+date: 2018-11-13T16:32:30+09:00
+weight: 68
+draft: true
+---
+
+### Service Mesh
+
+> A service mesh is a dedicated infrastructure layer for handling `service-to-service communication`. It’s responsible for the reliable delivery of requests through the complex topology of services that comprise a modern, cloud native application
```

Comment:
using backticks to emphasise should be replaced with `*` or `**` depending on if you prefer bold or italics.

Reply:
have replaced backticks to **